Samples/Opc.Ua.Sample/Base/DataChangeMonitoredItem.cs

```diff
@@ -700,12 +712,21 @@ public bool Publish(OperationContext context, Queue<MonitoredItemNotification> n
                     DataValue value = null;
                     ServiceResult error = null;
 
+                    int publishedItems = 0;
                     while (m_queue.Publish(out value, out error))
```

Comment:
Does this confirm to spec statement : "If a value is queued for a data MonitoredItem, the next value in the queue is sent in the Publish response." ? I must admit this statement a bit ambiguous.